manifests/config.pp

```diff
@@ -126,10 +126,6 @@
     $foreman_socket_override = template('foreman/foreman.socket-overrides.erb')
 
     if $foreman::ipa_authentication {
-      unless fact('foreman_ipa.default_server') {
-        fail("${facts['networking']['hostname']}: The system does not seem to be IPA-enrolled")
```

Comment:
I wonder what this guard originally was about?

Worst that can happen without it is that `ipa-getkeytab` below fails?

Reply:
Previously it would guaranteed generate an incorrect command if that fact was missing. However, we can't really reliably detect it (DNS records can be used for example). So you're right that ipa-getkeytab could fail during runtime and exec resources don't log well in our installer (happens at the info level and to make the installer quieter, info is only sent to `/var/log`). However, I do think that fully supporting all possible registrations is more important than perfect error handling in this case.